src/components/selection-list/selection-list.tsx

```diff
@@ -0,0 +1,136 @@
+import React = require('react');
+import ReactDOM = require('react-dom');
+import {SBComponent, SBStateless} from 'stylable-react-component';
+import {root} from 'wix-react-tools';
+import style from './selection-list.st.css';
+
+export const divider = {};
+
+function renameKeys(data: any, schema: any) {
```

Comment:
Maybe this one? https://basarat.gitbooks.io/typescript/content/docs/types/index-signatures.html